proxy/proxy.go

```diff
@@ -66,12 +87,18 @@ func (p *Proxy) identifyRequestsforGeneratedTracer(d []byte, method string) {
 			req.RawRequest = dump
 			req.RequestMethod = method
 
-			err = store.DB.Save(&req).Error
+			jsonData, _ := json.Marshal(req)
+
+			req, err := http.NewRequest("PATCH", "http://"+configure.Current.TracyServer.Addr()+"/api/tracy/tracers", bytes.NewBuffer(jsonData)) //Change this over to https when we do that. Well it would be better to make it a config
```

Comment:
Oooo. I see why you have request now. We might need a different API for this one and the one that is in master. This modifies the request data structure and the other one I was thinking of just modifies individual tracers. But this makes sense now, I forgot about this DB call in the proxy. Might also be a good to look at #85 because this is where it happens. Seems like when two tracers come in, there is a race condition where their tracer requests overwrite each other. Might be nice to store all requests that have that tracer string since we can't really tell. Or maybe just hold on to that for now and we can deal with it later.